cylc/flow/workflow_files.py

```diff
@@ -963,7 +963,7 @@ def infer_latest_run(
     if path.name == WorkflowFiles.RUN_N:
         runN_path = path
         if warn_runN:
-            LOG.warning(
+            LOG.info(
```

Comment:
No need to demote this?

Would only interact with the warning triangles on startup if `runN` is explicitly specified to `cylc play`?

Reply:
This is a warning because we expect the user to spot this and to take action (i.e. don't specify the runN bit).

If this is at the info level, it won't be spotted and the user won't take action.

I don't think why we would want to demote this?

Reply:
>  think it should not be a warning - it's just informing the user that its OK to omit the run number

Not quite.

This isn't telling you that it's unnecessary to specify the run name (we don't log that at all).

This is telling you that it's unnecessary to specify `runN` as the run name.

Reply:
Reverted. Not interested in courting controversy.

Reply:
Unresolving this as I just saw @oliver-sanders response to my comment. 

OK so I guess I didn't read carefully enough and misunderstood the text as saying you don't need to specify the run number at all.

If the warning is actually just triggered by use of the literal "runN" then I agree it should be a warning.

However, can we tweak the text to make it clear that it means the literal "runN" string, not say "run2" etc. ?

Reply:
OK, actually I'm happy with that wording, in context.